### PR TITLE
portage: fixes + added v1.1.*

### PR DESCRIPTION
**Pull request by junghans**
/cc @ddaniel @angelaherring @cmsquared @cferenba @nray

### CODE REVIEW

File: var/spack/repos/builtin/packages/portage/package.py [hunk 1]
```diff
@@ -34,20 +34,26 @@ class Portage(CMakePackage):
...
     def cmake_args(self):
         options = ['-DENABLE_UNIT_TESTS=ON', '-DENABLE_APP_TESTS=ON']
 
         if '+mpi' in self.spec:
             options.extend([
                 '-DENABLE_MPI=ON',
-                '-DENABLE_MPI_CXX_BINDINGS=ON'
+                '-DENABLE_MPI_CXX_BINDINGS=ON',
+                '-DCMAKE_CXX_COMPILER=%s' % self.spec['mpi'].mpicxx,
+                '-DCMAKE_C_COMPILER=%s' % self.spec['mpi'].mpicc,
```
Comment (junghans) [inline var/spack/repos/builtin/packages/portage/package.py]: @cmsquared workaround for laristra/portage#8

Comment (cmsquared) [reply var/spack/repos/builtin/packages/portage/package.py]: Looks good to me.  This is fixed upstream and will be unnecessary in 1.1.2.

Comment (junghans) [reply var/spack/repos/builtin/packages/portage/package.py]: Thanks, I will put this in a `if self.spec.satisfies('@:1.1.1'):` when adding 1.1.2.

**Comment by junghans** [comment]
@scheibelp

**Comment by junghans** [comment]
@tgamblin

**Pull request merged**